app/src/modules/app/app_object.cddl

```diff
@@ -0,0 +1,41 @@
+; Define the basic structure of the JSON object
+app-object = {
+    "nrfcloud_mqtt_topic_prefix": tstr,
+    "pairing": pairing-type,
+    "lwm2m": lwm2m-map
+}
+
+; Define the pairing object with nested topics
+pairing-type = {
+    "state": tstr,
+    "topics": {
+        "d2c": tstr,
+        "c2d": tstr
+    }
+}
+
+lwm2m-map = {
+  "14240:1.0": lwm2m_object_1,
```

Comment:
I believe we are free to name these `lwm2m_object_1` and `lwm2m_object` types to something more meaningful.  Say, `location_config` and `led_config`